### ISSUE TITLE
Remove link to Browse > Indicators in the top nav menu -- and anywhere else

### ISSUE THREAD

**Issue by jennyhavah** [mercycorps/TolaActivity#980]
Since home and program pages are loading quickly, we see little benefit to continue to maintain the master-list-of-indicators page. Particularly after we React-ify the program page, it'll be a pain to maintain in parallel.

# Acceptance criteria
The Browse menu in the top nav no longer contains "Indicators" as an option

I recommend also doing a search through the code for any other links to that page that may remain. For example, I just discovered one on the Add indicator page. (I've got that instance covered in #981, but there could be others I've missed.)

**Comment by jennyhavah** [comment]
@KennyMonster Turns out we actually do need to remove the breadcrumb link on the full page version of the indicator form, because it currently throws a "page not found" error.

![image](https://user-images.githubusercontent.com/33670923/50353853-5ffaff80-04fe-11e9-944c-6607e8409baa.png)

**Comment by KennyMonster** [comment]
Bread crumbs were also removed on "Indicator Plan" page. The program name was made clickable instead to allow navigation to the program page if desired.

**Comment by jennyhavah** [comment]
Regarding the program name on the Indicator Plan, @KennyMonster Do you know why the program name is appearing multiple times, and if that has anything to do with this ticket or is a separate issue? (It is not doing this on Production.)

Here's an example:

![image](https://user-images.githubusercontent.com/33670923/50499326-a4eecc80-09fd-11e9-9e75-bf1d929fcab8.png)

**Comment by KennyMonster** [comment]
@jennyhavah The format currently being displayed is "{{ GAIT ID }} - {{ Program Name }}". In the case of the QA programs, I think the GAID ID is being set to the same string as the program name, hence the duplication.

An example of an AF program with a real GAIT ID:

![screen shot 2018-12-28 at 9 14 45 am](https://user-images.githubusercontent.com/1278224/50522731-4c611300-0a81-11e9-90e3-3538b4759e07.png)
